x-pack/plugins/telemetry_collection_xpack/schema/xpack_plugins.json

```diff
@@ -5115,6 +5115,34 @@
               }
             }
           }
+        },
+        "resources_stats": {
+          "type": "array",
```

Comment:
Given that there's already a [validated POC](https://github.com/elastic/security-team/issues/4978#issuecomment-1359126442) for handling a list format in `edn`, it's ok to go ahead and use an array here.